index.js

```diff
@@ -56,10 +56,16 @@ function watch(glob, options, cb) {
 
   if (typeof cb === 'function') {
     var fn = debounce(onChange, opt.delay, opt);
-    watcher
-      .on('change', fn)
-      .on('unlink', fn)
-      .on('add', fn);
+
+    if (!opt.events) {
```

Comment:
this normalization should happen near the beginning (after the `var opt = ` declaration).  There should be an additional check after it for `Array.isArray(opt.events)` and if it isn't an array, it should wrap it in one (`opt.events = [opt.events]`